scala3doc-testcases/src/tests/specializedSignature.scala

```diff
@@ -0,0 +1,11 @@
+package tests
+
+package specializedSignature
+
+import scala.{specialized}
+
+trait AdditiveMonoid[@specialized(Int, Long, Float, Double) A]
+{
+  def a: A
```

Comment:
can we also add a case where `@specialized` is added to a function?

Reply:
Done